extensions/arc/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Improvements] - 2024-09-02
```

Comment:
```suggestion
## [Improvements] - {PR_MERGE_DATE}
```

Reply:
Ah, was wondering how to do this! Thanks